includes/modules/payment/paypalwpp.php

```diff
@@ -686,6 +686,9 @@ function install() {
 
     $db->Execute("insert into " . TABLE_CONFIGURATION . " (configuration_title, configuration_key, configuration_value, configuration_description, configuration_group_id, sort_order, date_added) values ('PayPal Merchant ID', 'MODULE_PAYMENT_PAYPALWPP_MERCHANTID', '', 'Enter your PayPal Merchant ID here. This is used for the more user-friendly In-Context checkout mode. You can obtain this value by going to your PayPal account, clicking on Profile and navigating to the My Business Info section; You will find your Merchant Account ID on that screen. A typical merchantID looks like FDEFDEFDEFDE11.', '6', '25', now())");
     $db->Execute("insert into " . TABLE_CONFIGURATION . " (configuration_title, configuration_key, configuration_value, configuration_description, configuration_group_id, sort_order, set_function, date_added) values ('Use InContext Checkout?', 'MODULE_PAYMENT_PAYPALWPP_CHECKOUTSTYLE', 'InContext', 'PayPal now offers a newer friendlier InContext (in-page) checkout mode (Requires that you enter your MerchantID in the Merchant ID Setting above). Or you can use the older checkout style which offers Pay Without Account by default but with a full-page-redirect.', '6', '25', 'zen_cfg_select_option(array(\'InContext\', \'Old\'), ', now())");
+    $db->Execute("insert into " . TABLE_CONFIGURATION . " (configuration_title, configuration_key, configuration_value, configuration_description, configuration_group_id, sort_order, set_function, date_added) values ('InContext Button Color?', 'MODULE_PAYMENT_PAYPALWPP_INCONTEXTCOLOR', 'gold', 'When PayPal\'s newer friendlier InContext (in-page) checkout mode is enabled PayPal offers different colors for the PayPal button. Select the color of the button to use.', '6', '25', 'zen_cfg_select_option(array(\'gold\', \'blue\', \'silver\'), ', now())");
+    $db->Execute("insert into " . TABLE_CONFIGURATION . " (configuration_title, configuration_key, configuration_value, configuration_description, configuration_group_id, sort_order, set_function, date_added) values ('InContext Button Size?', 'MODULE_PAYMENT_PAYPALWPP_INCONTEXTSIZE', 'small', 'When PayPal\'s newer friendlier InContext (in-page) checkout mode is enabled PayPal offers various PayPal button sizes. Select the size of the button to use.', '6', '25', 'zen_cfg_select_option(array(\'small\', \'tiny\', \'medium\'), ', now())"); 
+    $db->Execute("insert into " . TABLE_CONFIGURATION . " (configuration_title, configuration_key, configuration_value, configuration_description, configuration_group_id, sort_order, set_function, date_added) values ('InContext Button Shape?', 'MODULE_PAYMENT_PAYPALWPP_INCONTEXTSHAPE', 'small', 'When PayPal\'s newer friendlier InContext (in-page) checkout mode is enabled PayPal the PayPal button can take different shapes. Select the button shape to use.', '6', '25', 'zen_cfg_select_option(array(\'pill\', \'rect\'), ', now())"); 
```

Comment:
So, this was a last minute feature add (bell or whistle in my opinion) with the thought of a few things. First could completely omit any such designations in the original file as the values provided are the PayPal defaults (at this time), but also trying to think about all of the discussions in the forum about presentation and ability to modify aspects to suit. So I went all out with configurable menu options which brought to new ZC users the option to easily locate and then modify the setting(s) for the selection(s). I feel that at the moment (though pure speculation) that talking about the 1 in 500 end users, but... also not totally thinking that the end user is the one to likely adjust/consider the option as much as the one that is installing/upgrading the site. I do know that a test site on which this was applied that the default PP gold button looks terrible, the blue works but the silver makes it almost disappear but really matches the template (but it doesn't pop).

By moving all calls to this formatting into a shared file, I was trying to eliminate any such use, but then got to thinking about the DIYers and was trying to settle on a sort of happy medium considering PP seemed to think this "feature" of at least all three options to be necessary. 

Now that said, if you're thinking to just move color to a definition or similar and keep the other two as options, I could go with that and leave a note about the current potential other "styles". It all becomes mute if the shared file is brought into the jscript_ in order to support various representations of the PayPal button.

Essentially, while I'm satisfied that there is working code and believe that ZC needs anything it can get to regain the traction lost in the last 2 years or so. Don't know if it's a culture shift (generationally related) or if there is/was some ruffled feathers, but ZC in recent e-commerce reviews doesn't seem to even show. When commenters bring it up, the response is often not positive. :(